src/api/index.js

```diff
@@ -36,6 +38,23 @@ const search = searchTerm => new Promise((resolve, reject) => {
   });
 });
 
+const updateConfig = () => new Promise((resolve, reject) => {
```

Comment:
Can you expand on what this function is doing? You should also be able to use your new `getAllPlugins()` function below instead of calling `fs.readdir` here again.